interop/src/test/scala/InteropTest.scala

```diff
@@ -43,7 +62,7 @@ final class InteropTest extends AnyWordSpecLike with Matchers with TestLogCaptur
     val logged = capturedLog4sOutOf { logger =>
       import log.effect.interop.log4cats._
 
-      implicit val buildMessageLogger =
+      implicit val buildMessageLogger: SelfAwareStructuredLogger[IO] =
```

Comment:
Scala 3 again:
```
[error] -- [E124] Cyclic Error: /Users/jsirocchi/code/log-effect/interop/src/test/scala/InteropTest.scala:48:33
[error] 48 |        Slf4jLogger.fromSlf4j[IO](logger).unsafeRunSync()
[error]    |                                 ^
[error]    |value buildMessageLogger needs result type because its right-hand side attempts implicit search
```